### ISSUE TITLE
Request: scroll offset

### ISSUE THREAD

**Issue by ninjaasmoke** [ryanhefner/react-scroll-trigger#146]
How to trigger the onEnter after a certain offset in either direction? Thankyou!

**Comment by lancesnider** [comment]
Yeah, that'd be handy for things like determining when the element reaches the top of the screen.
